### DISCUSSION TITLE
Newbie is trying to train .... and does not able to ...

### DISCUSSION THREAD

**Discussion by Iaurkano64** [buliasz/tesstrain-windows-gui#7] [Q&A] [answered]
Hi buliasz, 

i'm starting to work with tesseract few days ago, and i found your repo to help to train.

Firstly, thank's very much for your work.

Secondly : I met a issue when i launch the "Train process" using your GUI. As i don't find any no doc, it's not so easy to understand what's happen.

Here after the error message : 

![image](https://user-images.githubusercontent.com/61876101/168043539-7da58594-cd1a-4396-8b01-55a4c961211a.png)

i'm searching through the web, with no result.

Thank's for your help/support.

**Comment by buliasz** [comment, marked as answer]
Hi @Iaurkano64 ,
It seems not related to GUI. 
Can you try copying this message above (pressing Ctrl+C when the message window is activated copies all the message text to clipboard) and try executing the command in your command line to verify if the result is the same.
If it is the same you'll need to report it on the [main tesstrain repo](https://github.com/tesseract-ocr/tesstrain), as those guys are responsible for developing the `unicharset_extractor` tool.
Please let me know results.

**Reply by Iaurkano64** [reply]
Hi, Buliasz,

But the fact is :  it's impossible do copy the dialog box contents, that's why i put the screen capture in my message not only the text message.

i try to write manually the command line.

Regards.

**Reply by buliasz** [reply]
https://www.youtube.com/watch?v=7tn6Scl8Uk4

**Reply by Iaurkano64** [reply]
> https://www.youtube.com/watch?v=7tn6Scl8Uk4

i didn't know this tips...

Thank's

**Reply by Iaurkano64** [reply]
question posted in the "main tesstrain repo"

[here]( https://github.com/tesseract-ocr/tesstrain/issues/306)

Regards.

**Reply by buliasz** [reply]
Responded. I think you've changed the `all-gt` file name because GUI do not create `all-gt.txt` but `all-gt`.

**Reply by Iaurkano64** [reply]
you're right , and it's not possible to opload in the forum a file without extension.
